packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -809,10 +806,13 @@ class EditorPage {
 	}
 
 	async enterCaptionToSelectedImageBlock( caption, clear = true ) {
-		const imageBlockCaptionField = await this.driver.elementByXPath(
+		const imageBlockCaptionButton = await this.driver.$(
 			'//XCUIElementTypeButton[starts-with(@name, "Image caption.")]'
 		);
-		await imageBlockCaptionField.click();
+		await imageBlockCaptionButton.click();
+		const imageBlockCaptionField = await imageBlockCaptionButton.$(
+			'//XCUIElementTypeTextView'
+		);
```

Comment:
The[ `clearValue`](https://webdriver.io/docs/api/element/clearValue/) method did not succeed on the caption Button element. Hence, the utility was expanded to instead [type into](https://github.com/WordPress/gutenberg/blob/90d8bb9ac5835b2cefbc0db245f088705db77541/packages/react-native-editor/__device-tests__/helpers/utils.js#L218-L220) the TextView.